sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+    -- selecting the events we need to compute on
+    AND (
+      (category = 'newtab.search' AND name IN ('issued'))
+      OR (category = 'newtab.search.ad' AND name IN ('click'))
+      OR (
+        category = 'pocket'
+        AND name IN (
+          'click',
+          'impression',
+          'dismiss',
+          'thumb_voting_interaction',
+          'thumb_voting_interaction'
+        )
+      )
+      OR (category = 'topsites' AND name IN ('click', 'impression', 'dismiss'))
+      OR (
+        category = 'newtab'
+        AND name IN (
+          'opened',
+          -- weather
+          'weather_change_display',
+          'weather_open_provider_url',
+          'weather_location_selected',
+          -- wallpaper
+          'wallpaper_click',
+          'wallpaper_category_click',
+          'wallpaper_highlight_cta_click',
+          'wallpaper_highlight_dismissed',
+          -- topic_selection
+          'topic_selection_open',
+          'topic_selection_dismiss',
+          'topic_selection_topics_saved',
+          -- section
+          'sections_block_section',
+          'sections_follow_section',
+          'sections_unblock_section',
+          'sections_unfollow_section',
+          'inline_selection_click'
+        )
+      )
+    )
+)
+SELECT
+  submission_date,
+  client_id,
+  mozfun.map.get_key(event_details, 'newtab_visit_id') AS newtab_visit_id,
+  -- ANY_VALUE: because we do not expect those fields to differ within a visit
+  ANY_VALUE(app_version) AS app_version,
+  ANY_VALUE(os) AS os,
+  ANY_VALUE(channel) AS channel,
+  ANY_VALUE(locale) AS locale,
+  ANY_VALUE(country) AS country,
+  ANY_VALUE(homepage_category) AS homepage_category,
+  ANY_VALUE(newtab_category) AS newtab_category,
+  ANY_VALUE(organic_content_enabled) AS organic_content_enabled,
+  ANY_VALUE(sponsored_content_enabled) AS sponsored_content_enabled,
+  ANY_VALUE(sponsored_topsites_enabled) AS sponsored_topsites_enabled,
+  ANY_VALUE(organic_topsites_enabled) AS organic_topsites_enabled,
+  ANY_VALUE(newtab_search_enabled) AS newtab_search_enabled,
+  LOGICAL_OR(is_default_ui) AS is_default_ui,
+  LOGICAL_OR(event_category = 'newtab' AND event_name = 'opened') AS is_newtab_opened,
+  LOGICAL_OR(event_category = 'newtab.search' AND event_name = 'issued')
+  AND LOGICAL_OR(is_default_ui) AS is_search_issued,
+  LOGICAL_OR(event_category = 'newtab.search.ad' AND event_name = 'click')
+  AND LOGICAL_OR(is_default_ui) AS is_search_ad_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click', 'dismiss', 'thumb_voting_interaction', 'thumb_voting_interaction')
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_content_interaction,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('click'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_click,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('impression'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_impression,
```

Comment:
Consolidate into one `LOGICAL_OR` unless there is a reason not to, see comment on line 95

Reply:
We cannot consolidate into one. We want to `AND` the result of each of those `LOGICAL_OR` aggregations that happen for different event types

Reply:
Why wouldn't `LOGICAL_OR(event_category = 'pocket' AND event_name IN ('impression') AND is_default_ui)` work here?

Reply:
I have explained it [here](https://github.com/mozilla/bigquery-etl/pull/7362/files#r2093337681)